src/TailscaleManager.hs

```diff
@@ -154,18 +153,11 @@ logDiff prevRoutes newRoutes = do
     routesToRemove  = S.difference prevRoutes newRoutes
     routesUnchanged = S.intersection prevRoutes newRoutes
 
--- |Compute how much the smaller the new set is vs old.
---
--- >>> shrinkRatio ["1.1.1.1/32", "2.2.2.2/32"] ["1.1.1.1/32"]
--- 0.5
-shrinkRatio :: Foldable t
-            => t a  -- ^ Old set
-            -> t a  -- ^ New set
-            -> Double    -- ^ Shrink ratio
+-- | Compute how much smaller the new set is vs old.
+shrinkRatio :: Foldable t => t a -> t a -> Double
```

Comment:
Please keep the haddock comments on the type args, those are actually parsed when generating code docs.

Reply:
sure, resolved :)